.rubocop.yml

```diff
@@ -29,6 +29,10 @@ Metrics/AbcSize:
   Exclude:
     - lib/pdc/config.rb
 
+Metrics/ModuleLength:
```

Comment:
Is this changing the ordering?
If so, we can make it in another patch.

Reply:
This is just for resolving the conflict, thanks for reminding that, will change back